frame/nfts/src/impl_nonfungibles.rs

```diff
@@ -135,6 +135,21 @@ impl<T: Config<I>, I: 'static> Inspect<<T as SystemConfig>::AccountId> for Palle
 	}
 }
 
+impl<T: Config<I>, I: 'static> InspectRole<<T as SystemConfig>::AccountId> for Pallet<T, I> {
+	/// Returns `true` if `who` is the issuer of the `collection`.
```

Comment:
You dont need to doc the implementation of trait functions.

Reply:
yeah strictly speaking, they will come already pre-documented with whatever the docs on the actual trait method are, so 99.99999% of the time you just want to use what it already has